src/tests/ftest/control/config_generate_run.py

```diff
@@ -4,6 +4,8 @@
   SPDX-License-Identifier: BSD-2-Clause-Patent
 '''
 
+import os
+
```

Comment:
isort would complain unless the space was here for some reason

Reply:
It's because `os` is internal and `yaml` is 3rd-party (pip installed or dnf isntalled) so isort wants those group separately